### ISSUE TITLE
Missing syntax error for comma after functional update syntax

### ISSUE THREAD

**Issue by Veykril** [rust-lang/rust-analyzer#15989]
```rs
Foo {
    ..bar,
}
```
The comma here is ill placed, but we do not report an error on it.
